lib/Model.php

```diff
@@ -460,12 +460,15 @@ public function assign_attribute($name, $value)
 		}
 
 		// convert php's \DateTime to ours
-		if (!($value instanceof DateTime) && $value instanceof \DateTime)
-			$value = new DateTime($value->format('Y-m-d H:i:s T'));
+		if (is_object($value) && get_class($value) === 'DateTime') {
```

Comment:
The double `instanceof` has been introduced just recently, and according to a benchmark the `instanceof` was performing better than `get_class` with string comparison.

See that conversation here: https://github.com/jpfuentes2/php-activerecord/pull/531#issuecomment-202779494